netlify/build-plugins/netlify-plugin-storyblok-redirects/index.js

```diff
@@ -0,0 +1,160 @@
+import StoryblokClient from 'storyblok-js-client';
+import dotenv from 'dotenv';
+
+// Call defaults.
+dotenv.config();
+
+/**
+ * Sanitize the redirect code to ensure it is a valid HTTP status code.
+ * @param {string} code
+ * @returns {number}
+ */
+const sanitizeRedirectCode = (code) => {
+  // Ensure code is a number and in one of 301, 302, 303, 307, 308
+  const statusCode = parseInt(code, 10);
+  if ([301, 302, 303, 307, 308].includes(statusCode)) {
+    return statusCode;
+  }
+  return 301;
+};
+
+/**
+ * Double-slash escape characters in a string.
+ * @param {*} str
+ * @returns {string}
+ */
+const sanitizeSourcePath = (str) => {
+  const parts = str.split('?');
+  const path = parts[0];
+  // Replace '*' with '(.*)' to match dynamic routes
+  return path.replace(/\*/g, '(.*)');
+};
+
+/**
+ * Extract query parameters from a URL.
+ * @param {string} url
+ * @returns {Array}
+ */
+const extractQueryParameters = (url) => {
+  // Parse the URL and get the query string
+  const urlObj = new URL(url, 'http://example.com'); // base URL needed for relative paths
+  const queryParams = new URLSearchParams(urlObj.search);
+
+  // Create an array to hold the query parameter objects
+  const queryArray = [];
+
+  // Iterate through each query parameter and add it to the array in the required format
+  queryParams.forEach((value, key) => {
+      queryArray.push({
+          type: 'query',
+          key: key,
+          value: value,
+      });
+  });
+
+  return queryArray;
+};
+
+/**
+ * Get redirects from Storyblok.
+ * @returns {Promise<Array>}
+ */
+const getStoryblokRedirects = async (isProd, storyPath, apiKey, region) => {
+  // Set up the Storyblok client
+  let storyblokApi = new StoryblokClient({
+    accessToken: apiKey,
+    region: region || 'eu',
```

Comment:
@yvonnetangsu, 

This works by setting the default to `eu` here, but now the documentation is off. To set the `region` I would have expected you to set it in the `netlify.toml` file.